app/assets/javascripts/controllers/cloud_volume/cloud_volume_form_controller.js

```diff
@@ -258,7 +258,12 @@ ManageIQ.angular.app.controller('cloudVolumeFormController', ['miqService', 'API
     vm.cloudVolumeModel.size = data.size / 1073741824;
     vm.cloudVolumeModel.cloud_tenant_id = data.cloud_tenant_id;
     vm.cloudVolumeModel.volume_type = data.volume_type;
-    vm.cloudVolumeModel.availability_zone_id = data.availability_zone.ems_ref;
```

Comment:
We were setting the `availability_zone_id` equal to the `ems_ref`...?  @himdel @skateman ?

Reply:
@agrare I also didnt understand it, I was planning on asking you/@skateman  about it

Reply:
wat? :open_mouth:

Reply:
I guess the API can work with both? cc @abellotti

Reply:
The code comes from https://github.com/ManageIQ/manageiq-ui-classic/pull/950 which is a while ago.. could it be relying on a specific provider behaviour that's no longer universal?

Reply:
I don't think so, we're working on a react rewrite of this form and while testing the old one, I haven't found any irregularities...